lib/libv8/node/version.rb

```diff
@@ -1,7 +1,7 @@
 module Libv8; end
 
 module Libv8::Node
-  VERSION = '16.19.0.0'.freeze
+  VERSION = '16.19.0.1'.freeze
```

Comment:
Please don't bump the version in the PR 🙏

Reply:
reverting